.delivery/build-cookbook/LICENSE

```diff
@@ -0,0 +1,3 @@
+Copyright 2015 The Authors
```

Comment:
Let's make this Apache 2 or just remove it and assume that the build cookbook is under the project license.